functions/src/firebase-functions-types.ts

```diff
@@ -1,12 +1,33 @@
+import type { LobbySettings } from "./firestore-types/lobby";
+
 export type LobbyRequest = { code: string };
 
+export type LobbySettingsRequest = {
+  code: string;
+  lobbySettings: LobbySettings;
+};
+
 export type LobbyCreationResponse = { code: string };
 
 export function isLobbyRequest(data: unknown): data is LobbyRequest {
   // will only return true if the data is an object with a code property and string
   return data != null && typeof data === "object" && "code" in data && typeof (data as LobbyRequest).code === "string";
 }
 
+export function isLobbySettingsRequest(data: unknown): data is LobbySettingsRequest {
+  return (
+    data != null &&
+    typeof data === "object" &&
+    "code" in data &&
+    "lobbySettings" in data &&
+    typeof (data as LobbySettingsRequest).code === "string" &&
+    typeof (data as LobbySettingsRequest).lobbySettings.catfishAmount === "number" &&
+    typeof (data as LobbySettingsRequest).lobbySettings.promptTime === "number" &&
+    typeof (data as LobbySettingsRequest).lobbySettings.chatTime === "number" &&
+    typeof (data as LobbySettingsRequest).lobbySettings.voteTime === "number"
+  );
+}
```

Comment:
you should make tests for this function.
look at `functions/tests/functions-types/avatar.test.ts` for an example

Reply:
there are some security issues that I think will become clear to you when you make the tests